eureka-webapp/src/main/resources/Messages.properties

```diff
@@ -49,7 +49,7 @@ phenotypetype.FREQUENCY.description=For defining a disease, finding or patient c
 phenotypetype.VALUE_THRESHOLD.displayName=Value Threshold
 phenotypetype.VALUE_THRESHOLD.description=For defining clinically significant thresholds on the value of an observation.
 passwordChange.error.internalServerError=Error while changing password.
-deletePhenotype.error.internalServerError=Error trying to delete phenotype. The phenotype is in use. There is a problem with Eureka!.
+deletePhenotype.error.internalServerError=Error trying to delete phenotype. The phenotype is in use. There is a problem with Eureka!
```

Comment:
For "Error deleting phenotype '{0}' ", we have similar functions in front-end, at editor_home.jsp 

> error: function (data, statusCode, errorThrown) {
> 							var content = 'Error while deleting &quot;' + displayName.trim() + '&quot;. ' + data.responseText + '. Status Code: ' + statusCode;
This part of code will print error deleting phenotype name, so no need to add that code in java or in messages.properties file. 

However, I do change the content of "deletePhenotype.error.internalServerError" a little bit, because when this error shows up, it is not because the phenotype is in use, but internal server error. So it looks like this now:
<img width="613" alt="screen shot 2016-05-12 at 1 33 50 pm" src="https://cloud.githubusercontent.com/assets/5263886/15224358/e242c420-1846-11e6-8b5d-8917ef386e44.png">